### PR TITLE
[11.0] Company also need a fax number

### PR DESCRIPTION
**Pull request by satriani-vai**

### CODE REVIEW

**Comment by bealdav** [comment]
Thanks a lot

Not sure it's required because, res.company have a m2o to res.partner (partner_id)

**Comment by satriani-vai** [comment]
It means a fax number for the own company and not for the person.

**Comment by bealdav** [comment]
Yes but partner in company is not a contact. Just define it as a company in partner form

**Comment by NL66278** [comment]
@satriani-vai Please close your previous PR's that had the same intention as this one.

File: partner_fax/models/res_company.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+
+    _inherit = 'res.company'
+
+    fax = fields.Char()
```
Comment (JayVora-SerpentCS) [inline partner_fax/models/res_company.py:8]: I don't think this is needed. make a related field to partner_id like we have phone and email.
Save redundancy.

Comment (rafaelbn) [reply partner_fax/models/res_company.py:8]: I think this is the same opinion as @bealdav said in https://github.com/OCA/partner-contact/pull/572#issuecomment-370159649 . Isn't it? :thinking:

Comment (JayVora-SerpentCS) [reply partner_fax/models/res_company.py:8]: yes, Similar, we can add field, but related type only.

Comment (rafaelbn) [reply partner_fax/models/res_company.py:8]: @satriani-vai thanks for this contribution, I think recommendation from @bealdav and @JayVora-SerpentCS is correct. Don't you think so?

Comment (satriani-vai) [reply partner_fax/models/res_company.py:8]: Yes it is. I would keep fax in all contact types.

Comment (rafaelbn) [reply partner_fax/models/res_company.py:8]: Hello @satriani-vai if you change this to ma a related field to partner_id, then @JayVora-SerpentCS and @bealdav could give you the :+1 for mergin :smiley: Could you make the change please? Thank you!

Comment (SalahAdDin) [reply partner_fax/models/res_company.py:8]: @satriani-vai What happened bro?

We are still waiting for this.

**Comment by SalahAdDin** [comment]
:+1:

**Review by chienandalu — COMMENTED** [review]
This PR is still pending of the simplest change. Is it going to be finished?

**Comment by emagdalenaC2i** [comment]
@pedrobaeza I think this PR should be closed because it is already merged in #547

**Comment by SalahAdDin** [comment]
Thanks man!

**Pull request closed without merging**